src/main.js

```diff
@@ -69,16 +86,31 @@ const headerElement = document.querySelector('.header');
 const mainElement = document.querySelector('.main');
 renderElement(headerElement, new UserStatusView().getElement(), RenderPosition.BEFOREEND);
 renderElement(mainElement, new MenuView(dataFilms).getElement(), RenderPosition.BEFOREEND);
-renderElement(mainElement, new SortView().getElement(), RenderPosition.BEFOREEND);
+dataFilms.length ? renderElement(mainElement, new SortView().getElement(), RenderPosition.BEFOREEND) : null;
 const filmsBoardComponent = new FilmsBoardView();
+const listTitle = new ListTitle();
+const listTitleEmpty = new ListTitleEmpty();
 renderElement(mainElement, filmsBoardComponent.getElement(), RenderPosition.BEFOREEND);
 // const filmsElement = mainElement.querySelector('.films');
 const filmListComponent = new FilmListView();
+// const replaceTitleToEmptyTitle = () => {
+//   filmListComponent.getElement().replaceChild(listTitle.getElement(), listTitleEmpty.getElement())
+// }
+// const replaceTitleEmptyToTitle = () => {
+//   filmListComponent.getElement().replaceChild( listTitleEmpty.getElement(),listTitle.getElement())
+// }
+if(dataFilms.length) {
+  renderElement(filmListComponent.getElement(), listTitle.getElement(), RenderPosition.AFTERBEGIN);
+}else {
+  renderElement(filmListComponent.getElement(),  listTitleEmpty.getElement(), RenderPosition.AFTERBEGIN);
+}
```

Comment:
А тут проще через тернарник
```suggestion
renderElement(filmListComponent.getElement(), dataFilms.length ? listTitle.getElement() : listTitleEmpty.getElement(), RenderPosition.AFTERBEGIN);
```